plopfile.js

```diff
@@ -0,0 +1,36 @@
+module.exports = (plop) => {
+    plop.setGenerator("exercise", {
+        description: "generate a new exercise with tests",
+        prompts: [
+            {
+                type: "list",
+                name: "concept",
+                message: "pick a JS concept",
```

Comment:
```suggestion
                message: "pick a JavaScript concept",
```
beginners might not know JS = JavaScript

Reply:
This is only seen by people adding exercises to the repo. :)